setup.py

```diff
@@ -82,7 +82,7 @@ def read_description(module_name):
     },
     entry_points={
         'console_scripts': [
-            'wawsync = wawsync.console:main',
+            'pyfilesync = pyfilesync.console:main',
```

Comment:
Może sam program który doda nasz pakiet nazwać `filesync` zamiast `pyfilesync`

Reply:
Możemy go tak nazwać, w sumie nawet w pip go nie ma.

Reply:
Nazwy pozmieniać tylko w setup.py?

Reply:
Akurat w pip będzie nazwa pakietu `pyfilesync` i niech tak zostanie. Mi chodzi tylko o program konsolowy który się doinstaluje po instalacji pakietu.

Reply:
W każdym razie może ja już to ogarnę przy robieniu tego programu ;)